zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -825,6 +837,26 @@ public void setInfo(Map<String, Object> info) {
     this.info = info;
   }
 
+  public String getVersion() {
+    String version = StringUtils.EMPTY;
+    if (info.get("version") == null) {
+      return version;
+    }
+    try {
+      version = (String) info.get("version");
+    } catch (ClassCastException e) {
```

Comment:
actually I may remove `version` info in this PR since haven't been used actively here. instead i'll add it as separate pr with more details